index.js

```diff
@@ -18,6 +18,7 @@
 exports.arrays = require('./lib/arrays.js');
 exports.base32hex = require('./lib/base32hex.js');
 exports.config = require('./lib/config.js');
+exports.mongo = require('./lib/mongo.js');
```

Comment:
Should we name this export differently, to disambiguate it from, say, the mongo driver? Perhaps `exports.mongoUtil`?

Reply:
There are only two hard things in Computer Science: cache invalidation and naming things.

-- the late Phil Karlton

Reply:
@pazaan Unless you feel strongly, I am inclined to decline this request.  I don't think it warrants cycling through all the clients. :)